src/main/java/com/mateacademy/springmvc/bpp/UserBeanPostProcessor.java

```diff
@@ -0,0 +1,28 @@
+package com.mateacademy.springmvc.bpp;
+
+import com.mateacademy.springmvc.entity.User;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.BeansException;
+import org.springframework.beans.factory.config.BeanPostProcessor;
+
+
```

Comment:
Should we add annotation @Component here?

Reply:
yes